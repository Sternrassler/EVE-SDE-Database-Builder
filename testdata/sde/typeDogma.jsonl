{"typeID":34,"dogmaAttributes":[{"attributeID":4,"value":0.0}],"dogmaEffects":[{"effectID":10,"isDefault":1}]}
{"typeID":35,"dogmaAttributes":[{"attributeID":4,"value":0.0}],"dogmaEffects":[{"effectID":10,"isDefault":1}]}
{"typeID":36,"dogmaAttributes":[{"attributeID":37,"value":100.0}],"dogmaEffects":[{"effectID":11,"isDefault":0}]}
