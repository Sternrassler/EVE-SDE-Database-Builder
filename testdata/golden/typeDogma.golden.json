[
  {
    "typeID": 34
  },
  {
    "typeID": 35
  },
  {
    "typeID": 36
  }
]